Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -37,8 +37,10 @@
      *
      * @param game
      */
-    public GameEngine(Game game) {
-        this.game = game;
+    public GameEngine(Session session) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): session [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)